plenum/server/replica.py

```diff
@@ -292,6 +292,13 @@ def __init__(self, node: 'plenum.server.node.Node', instId: int,
         # instance is
         self._primaryName = None  # type: Optional[str]
 
+        # PRE-PREPAREs timestamps stored by non primary replica to check
+        # obsolescence of incoming PrePrepares.
+        # Dictionary:
+        #   key: Tuple[pre-prepare, sender]
+        #   value: timestamp
+        self.pre_prepare_tss = {}
```

Comment:
Is it sufficient just to use 3PC key (view_no, pp_seq_no) as a key in this dict like in other dicts, so that it can be easily removed in GC?